src/Compilers/CSharp/Portable/Binder/Binder_Conversions.cs

```diff
@@ -166,13 +166,14 @@ private BoundExpression ConvertObjectCreationExpression(UnboundObjectCreationExp
                 // if the destination is nullable, in which case we
                 // target the underlying type e.g. `S? x = new();`
                 // is actually identical to `S? x = new S();`.
+                var conversion = new Conversion(ConversionKind.ImplicitNullable, Conversion.IdentityUnderlying);
                 expr = new BoundConversion(
                     node.Syntax,
                     operand: expr,
-                    conversion: new Conversion(ConversionKind.ImplicitNullable, Conversion.IdentityUnderlying),
+                    conversion: conversion,
                     @checked: false,
                     explicitCastInCode: isCast,
-                    conversionGroupOpt: null,
+                    conversionGroupOpt: new ConversionGroup(conversion),
```

Comment:
We don't want the ObjectCreation conversion to be picked up by nullable walker, so the the conversion group has to be backwards.. (I noticed there's a plan to remove this https://github.com/dotnet/roslyn/issues/27163)

Reply:
I'm not sure what backwards means here? Honestly, we can probably close #27163, I don't think we'll be removing ConversionGroup.

Reply:
Talking with @cston, I'm going to close that issue. `ConversionGroup` is here to stay.